src/lib/task.js

```diff
@@ -400,7 +400,10 @@ class Task extends EventEmitter {
         Env: taskEnvToDockerEnv(env),
         HostConfig: {
           Privileged: privilegedTask,
-          ShmSize: 1800000000
+          ShmSize: 1800000000,
+          ExtraHosts: [
+            'localhost.localdomain:127.0.0.1', // for nss-try
+          ],
```

Comment:
This seems like the critical change, and seems fine (maybe link to the bug instead of just saying it's for nss-try? It seems pretty generic and other projects might want the functionality too)